titus-server-master/src/main/java/com/netflix/titus/master/mesos/kubeapiserver/direct/DefaultDirectKubeApiServerIntegrator.java

```diff
@@ -146,7 +150,14 @@ public void shutdown() {
                 metrics.terminateError(task, e, timer.elapsed(TimeUnit.MILLISECONDS));
 
                 if (e.getMessage().equalsIgnoreCase(NOT_FOUND) && task.getStatus().getState() == TaskState.Accepted) {
-                    sendEvent(PodEvent.onPodNotFound(task));
+                    sendEvent(PodEvent.onPodNotFound(task,
+                            TaskStatus.newBuilder()
+                                    .withState(TaskState.Finished)
+                                    .withReasonCode(TaskStatus.REASON_TASK_LOST)
+                                    .withReasonMessage("Pod terminate requested, but the pod is not found")
```

Comment:
minor: maybe `s/pod/task/` since this is an user facing message

Reply:
Here the task exists in Titus, but there is no pod. Maybe something like this:
`Task terminate requested, but its container is not found`